onnx/defs/function.h

```diff
@@ -141,6 +141,37 @@ class FunctionBuilder {
     return Add(node_txt, MakeAttribute(attr_name, attr_value));
   }
 
+  template <typename T>
```

Comment:
Wasn't this code present earlier? I thought you had something like this in an earlier PR. But it is not used in this PR, I think?

Reply:
Had added this as part of SkipNorm ops previously, added back in this PR for if else blocks. Think it could be a useful utility function to have anyways for the future but can remove if you think its not needed right now.